### PR TITLE
New package: Lapce.Lapce version 0.0.12

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58028)

### CODE REVIEW

File: manifests/l/Lapce/Lapce/0.0.12/Lapce.Lapce.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VC++2015-2022Redist-x64
+ReleaseDate: 2022-03-24
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/lapce/lapce/releases/download/v0.0.12/Lapce-windows.msi
+  InstallerSha256: D6A0208C531E0F75A3FBBD9939EB6EE30A6D6330D633E9B98F1A5DEEA48C0C0E
+  ProductCode: '{33BB89E1-F794-4048-8381-3237B7395092}'
```
Comment (OfficialEsco) [inline manifests/l/Lapce/Lapce/0.0.12/Lapce.Lapce.installer.yaml:18]:
```suggestion
  ProductCode: '{13DB46F9-9626-41A4-A654-B6C61E9F0AF6}'
```

**Review by OfficialEsco — COMMENTED** [review]
Different ProductCode?
```
Enter a pull request number: 58028

Manifest validation succeeded.
Found Lapce [Lapce.Lapce] Version 0.0.12
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/lapce/lapce/releases/download/v0.0.12/Lapce-windows.msi
  ██████████████████████████████  17.9 MB / 17.9 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : Lapce
Publisher     : Lapce
Version       : 0.0.12
ProductCode   : {13DB46F9-9626-41A4-A654-B6C61E9F0AF6}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{13DB46F9-9626-41A4-A654-B6C61E9F0AF6}
Uninstall     : winget uninstall --id "{13DB46F9-9626-41A4-A654-B6C61E9F0AF6}"
```

File: manifests/l/Lapce/Lapce/0.0.12/Lapce.Lapce.installer.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/l/Lapce/Lapce/0.0.12/Lapce.Lapce.installer.yaml:18]: The ProductCode from msi property differs from that from the registry, and both of them in each version differs from another. Maybe UpgradeCode is better in this scenario. 🤷

Comment (Trenly) [reply manifests/l/Lapce/Lapce/0.0.12/Lapce.Lapce.installer.yaml:18]: When I look at the MSI and the Registry they do match, and they are what Esco posted
image

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**